.gitignore

```diff
@@ -36,4 +36,4 @@ xmlrpc.php
 /themes
 
 # Ignore other actions
-actions/
+#actions/
```

Comment:
This should stay committed, if you want to add an action it can be added with `git add -f`, most of the times you will not add the "action" file to git, since it's most likely a single use.